tests/foreman/cli/test_import.py

```diff
@@ -481,25 +641,22 @@ def test_bz1160847_translate_macros(self):
             csv_writer.writerow(trans_header)
             csv_writer.writerow(trans_row)
 
-        # upload the files and remove them on local
         ssh.command('mkdir -p ~/.transition_data')
-        ssh.upload_file(file_name, os.path.basename(file_name))
         ssh.upload_file(
             trans_file, '.transition_data/' + os.path.basename(trans_file),
```

Comment:
`os.join('.transition_data/', os.path.basename(trans_file))`

also transition_data_file should give more meaning than trans_file.